### PR TITLE
bug: advanced payout

### PR DESCRIPTION
**Pull request by thelostone-mc**
- Ensure `tr` tag is placed inside `table` element
- code clean up
- ensure that `update_registry` is invoked when autcomplete is updated 

fixes: https://github.com/gitcoinco/web/issues/2380

Need an extra pair of eyes on testing this out 

x

### CODE REVIEW

**Comment by owocki** [comment]
just tested... heres my test video: http://bits.owocki.com/29c528a7c1ca/Screen%20Recording%202018-10-08%20at%2003.40%20PM.mov

bugs:
1. it says 'bounty stake' in the tx preview no matter what, regardless of whether i check the checkbox or not.  also, shouldn't we just remove this checkbox anyway?
2. i cannot add more than one github username to advanced payout.  or if i can, the payout preview is not updating..

after the above bugs are fixed... i can test the actual submission of the transactions

**Comment by Anish-Agnihotri** [comment]
@owocki 

Part of these issues I've fixed in #2365. Payout preview updating when you remove users, removing the bounty stake checkmark, etc. 

I'm currently refactoring the code according to suggestions from @thelostone-mc, @mbeacom, and @pinkiebell.

I'll make a fresh PR within the next hour to two.

**Comment by thelostone-mc** [comment]
@owocki addressed those comments 👍

**Comment by owocki** [comment]
@thelostone-mc the above two bugs are fixed, but now i [get a JS error when i try to payout](http://bits.owocki.com/1d4bc26bdc73/Screen%20Shot%202018-10-08%20at%204.17.32%20PM.png)

send.js:167 Uncaught TypeError: Cannot read property 'name' of null
    at sendTip (send.js:167)
    at sendTransaction (bulk_payout.js:121)
    at HTMLAnchorElement.<anonymous> (bulk_payout.js:146)
    at HTMLAnchorElement.dispatch (jquery.js:3)
    at HTMLAnchorElement.q.handle (jquery.js:3)

**Comment by owocki** [comment]
ahh nvm, that seems unrelated to this scope

**Comment by owocki** [comment]
its working!  yay!
butt0vcqjl04m

File: app/assets/v2/js/pages/bulk_payout.js [hunk 1]
```diff
@@ -187,15 +154,15 @@ $(document).ready(function($) {
...
+    var num_rows = $('#payout_table tbody').find('tr').length;
     var percent = num_rows <= 1 ? 100 : '';
     var denomination = $('#token_name').text();
     var amount = get_amount(percent);
     var html = `
       <tr>
         <td class="pl-0 pb-0">
           <div class="pl-0">
-            <select id="username" class="username-search custom-select" style="width: 100%; margin-left: auto; margin-right: auto;"></select>
+            <select id="username" onchange="update_registry()" class="username-search custom-select" style="width: 100%; margin-left: auto; margin-right: auto;"></select>
```
Comment (octavioamu) [inline app/assets/v2/js/pages/bulk_payout.js]: I think we should remove the Id to avoid id repetition

**Comment by octavioamu** [comment]
good clean up @thelostone-mc

File: app/assets/v2/js/pages/bulk_payout.js (hunk 1 above)
Comment (thelostone-mc) [reply app/assets/v2/js/pages/bulk_payout.js]: @Anish-Agnihotri mind addressing that in your PR ? 😅

Comment (Anish-Agnihotri) [reply app/assets/v2/js/pages/bulk_payout.js]: For sure, just threw that into the PR as well.

**Pull request merged**